funding/api.py

```diff
@@ -21,7 +21,8 @@ def api_proposals_get(status, cat, limit, offset):
         return 'error', 500
     return [p.json for p in proposals]
 
-@app.route('/api/1/convert/aeon-usd')
+
+@app.route('/api/1/convert/' + settings.COINNAME + '-usd')
```

Comment:
COINNAME -> COINCODE? Since COINCODE is defined in settings.

If COINNAME is new, include in settings.py_example?